socorro/external/es/super_search_fields.py

```diff
@@ -1767,11 +1756,10 @@ def apply_schema_properties(fields, schema):
         "query_type": "enum",
         "storage_mapping": {
             "fields": {
-                "plugin_filename": {"index": "analyzed", "type": "string"},
-                "PluginFilename": {"index": "analyzed", "type": "string"},
```

Comment:
if i'm reading the docs correctly, without `"path": "just_name"` this would have been accessed as `plugin_filename` or `plugin_filename.PluginFilename` and they would have the same result, but I expect we actually just want `plugin_filename` here, so this change removes `plugin_filename.PluginFilename`

same for `plugin_name.PluginName` below